### PR TITLE
Adding card "TT_hdamp_NNPDF31_13TeV_powheg.input"

### PR DESCRIPTION
**Pull request by yw5mj**
Adding powheg input card for TT_Mtt-1000toInf_TuneCP5_13TeV-powheg-pythia8 and TT_Mtt-700to1000_TuneCP5_13TeV-powheg-pythia8

### CODE REVIEW

**Comment by perrozzi** [comment]
why "Adding powheg input card for TT_Mtt-1000toInf_TuneCP5_13TeV-powheg-pythia8 and TT_Mtt-700to1000_TuneCP5_13TeV-powheg-pythia8" ?

**Comment by yw5mj** [comment]
This is the input card for the MCv2 RunIIFall17 samples TT_Mtt-1000toInf_TuneCP5_13TeV-powheg-pythia8 and TT_Mtt-700to1000_TuneCP5_13TeV-powheg-pythia8 with NNPDF3.1, these 2 samples are in part 1.

File: bin/Powheg/production/2017/13TeV/TT_hdamp/TT_hdamp_NNPDF31_13TeV_powheg.input [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+lhans1 306000
+lhans2 306000
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5  0.25 ! for not equal pdf sets
+ebeam1 6500    ! energy of beam 1
+ebeam2 6500    ! energy of beam 2
+qmass  172.5     ! mass of heavy quark in GeV
+facscfact 1    ! factorization scale factor: mufact=muref*facscfact 
+renscfact 1    ! renormalization scale factor: muren=muref*renscfact 
+#fixedscale 1    ! use ref. scale=qmass (default 0, use running scale)
```
Comment (perrozzi) [inline bin/Powheg/production/2017/13TeV/TT_hdamp/TT_hdamp_NNPDF31_13TeV_powheg.input]: please remove commented lines

**Comment by yw5mj** [comment]
Sorry for the late action. The commented lines are removed.

**Comment by perrozzi** [comment]
looks ok to me now

**Comment by efeyazgan** [comment]
hdamp should have been 1.379*mt=237.8775  to be consistent with other ttbar samples using CP5.

**Comment by yw5mj** [comment]
Sure, I was unsure about the hdamp parameter too. The old cards have 272.7225. I changed the value to 237.8775. Thanks.

**Pull request merged**